cmd/cli/upsert.go

```diff
@@ -58,4 +58,6 @@ func init() {
 	upsertCmd.Flags().DurationVar(&upsertArgs.MinRetryTime, "retry-min-time", time.Millisecond*200, "Minimum wait interval")
 	upsertCmd.Flags().DurationVar(&upsertArgs.MaxRetryTime, "retry-max-time", time.Second*30, "Maximum wait interval")
 	upsertCmd.Flags().IntVar(&upsertArgs.MaxRetryCount, "retry-max-count", 12, "Maximum number of retries before giving up")
+	upsertCmd.Flags().BoolVar(&upsertArgs.AppendGroups, "append-groups", false, "append to a existing group list")
+	upsertCmd.Flags().BoolVar(&upsertArgs.DontUpdateUsername, "dont-update-username", false, "if username exists, dont update it")
```

Comment:
Should we change this to `--update-username` with default being true instead of the double negative?
might be a bit less confusing

Reply:
Initially I had it that way but then I thought it might be confusing.
```
aws-auth upsert userroles --rolearn < rolearn > --username < username> --groups < group > 
```
would mean yes update username 

and 
```
aws-auth upsert userroles --rolearn < rolearn > --username < username> --groups < group > --update-username
```
would mean dont update username.

we could set `--update-username` to default false but then aws-auth isnt backwards compatible.

Its your call, more then happy to change it.

Reply:
Yeah, I think if we established the the default behavior is to overwrite username, it can be used with `--update-username=false` in order to avoid overwrite. 

e.g.

1) Username is overwritten (--update-username=true by default)
```
aws-auth upsert userroles --rolearn < rolearn > --username < username> --groups < group > 
```

2) Username is not overwritten by specifying `--update-username=false`
```
aws-auth upsert userroles --rolearn < rolearn > --username < username> --groups < group > --update-username=false
```

Perhaps down the road we can release a version with switching the default value from true to false if it makes more sense to not overwrite